apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -1152,9 +1152,9 @@ namespace MWPhysics
 
                 bool waterCollision = false;
                 if (effects.get(ESM::MagicEffect::WaterWalking).getMagnitude()
-                        && cell->getCell()->hasWater()
-                        && !world->isUnderwater(iter->first.getCell(),
-                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
+                        && cell->getCell()->hasWater())
+//                        && !world->isUnderwater(iter->first.getCell(),
+//                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
```

Comment:
We appreciate it that you want to help, but with a project of this size, it makes our job easier when everyone follows the same coding guidelines.

I bring this up because you ignored scrawl and mine's advice twice now, 17 days ago and again yesterday for now comments.

We would rather bounce the patch back to your with critique to make it better and to ease you into our workflow.

As for wasting your time, I'm sorry about that. We had no idea that your time was worth more than our time.